components/builder-api-proxy/habitat/config/nginx.conf

```diff
@@ -86,7 +86,9 @@ http {
                    '"$http_referer" "$http_user_agent" - $http_x_forwarded_for';
 
   upstream backend {
-    server {{sys.ip}}:{{bind.http.first.cfg.port}};
+    {{~#each bind.http.members as |member|}}
```

Comment:
Unfortunately this breaks our hosted service where we run them both together... we'll have to do a larger migration to a model where we run a single nginx reverse proxy running builder-api-proxy, and multiple backends running builder-api

Reply:
I can't get the two services running inside the same pod with our operator.

How about if I add a `backend-mode` option which defaults to the current logic (use `sys.ip`) but allows for a `load-balanced` mode which does this new logic.

Reply:
That seems reasonable - go for it